sdk/rust/core/src/token.rs

```diff
@@ -469,28 +369,6 @@ mod governance_packet_impl {
 
                     Action::ContractUpgrade { new_contract }
                 }
-                3 => {
-                    let ModifyBalance {
-                        sequence,
-                        chain_id,
-                        token_chain,
-                        token_address,
-                        kind,
-                        amount,
-                        reason,
-                    } = seq
-                        .next_element()?
-                        .ok_or_else(|| Error::invalid_length(3, &EXPECTING))?;
-                    Action::ModifyBalance {
-                        sequence,
-                        chain_id,
-                        token_chain,
-                        token_address,
-                        kind,
-                        amount,
-                        reason,
-                    }
-                }
                 v => {
                     return Err(Error::custom(format_args!(
                         "invalid value: {v}, expected one of 1, 2, 3"
```

Comment:
This needs to change to either "of 1, 2" or the error message should just be `"invalid value: {v}"`.

If the latter is chosen, we can change the same error in _accountant.rs_.